Makefile

```diff
@@ -163,6 +164,20 @@ download-binaries: kustomize go-bindata envtest controller-gen
 	curl -sSLO https://github.com/bats-core/bats-core/archive/v${BATS_VERSION}.tar.gz && tar -zxvf v${BATS_VERSION}.tar.gz && bash bats-core-${BATS_VERSION}/install.sh $(PWD)/ci-tools
 	rm -rf bats-core-${BATS_VERSION} v${BATS_VERSION}.tar.gz
 
+DEV_IMG=localhost:5000/gatekeeper-operator:dev
+.PHONY: kind-bootstrap-cluster
+kind-bootstrap-cluster: test-cluster dev-build
```

Comment:
Is this used anywhere or just for local development?

Reply:
just for local development?